### DISCUSSION TITLE
How to customize collections and module navigation visibility for each user role?

### DISCUSSION THREAD

**Discussion by agusibrahim** [directus/directus#14603] [Archive]
I didnt see that options again in roles and permissions (9.14.3 version). plz help

**Comment by licitdev** [comment]
You can customise collections visibility through `Settings -> Roles & Permissions`.
Collections with `READ` permissions allowed are visible to the role's users. 
Ref: https://docs.directus.io/configuration/users-roles-permissions/#configure-permissions

For module navigation visibility, please refer to https://github.com/directus/directus/discussions/10421 for more information.
You may wish to upvote the idea or look into [sponsored work](https://docs.directus.io/getting-started/support/#sponsored-work).
